### PR TITLE
Inspirationcard

### PR DESCRIPTION
**Pull request by Han-90**

### CODE REVIEW

File: src/components/Card/Card.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import {inspirations} from '../../data';
+import ArtistCard from '../ArtistCard/ArtistCard';
+
+import {InspirationCard} from './Card.styled';
+export default function Card() {
+	console.log(inspirations);
```
Comment (stefankorder) [inline src/components/Card/Card.js:6]: Please delete unnecessary console.log's

**Review by stefankorder — CHANGES_REQUESTED** [review]
Just one Suggestion.

File: src/components/Card/Card.js (hunk 1 above)
Comment (Han-90) [reply src/components/Card/Card.js:6]: thank you

Comment (Han-90) [reply src/components/Card/Card.js:6]: @stefankorder could you checkt it again please? I've solved the merge conflict and deleted the console.log my git hub actions are active now

**Pull request merged**